pkg/rpc/rpc.proto

```diff
@@ -1,7 +1,11 @@
 syntax="proto3";
 
+package longhorn.io.backing.image.manager.pkg.rpc;
```

Comment:
Too many layers... Should not be `longhorn.backingimagemanager.pkg.rpc`?

Reply:
Sounds good.